symfony/src/Form/MeasurementType.php

```diff
@@ -94,9 +94,11 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             } elseif ($type === 'date') {
                 unset($fieldOptions['scale']);
                 $class = Type\DateType::class;
-                $constraints[] = new Constraints\LessThanOrEqual([
-                    'value' => new \DateTime('today'),
-                    'message' => 'Date cannot be in the future'
+                $constraints[] = new Constraints\Range([
+                    'min' => new \DateTime('-6 months'),
+                    'max' => new \DateTime('today'),
+                    'minMessage' => 'Date cannot greater than six months in the past',
```

Comment:
Accidentally a word.

```suggestion
                    'minMessage' => 'Date cannot be greater than six months in the past',
```